src/game_manager.rs

```diff
@@ -78,7 +78,9 @@ impl GameManager {
         // TODO(orkaboy): detect level up screen
         if cmd.encounter_active {
             // Stop whatever we're doing and enter combat controller
-            context.gamepad.release_all();
+            for gamepad in context.gamepads.iter_mut() {
+                gamepad.release_all();
+            }
```

Comment:
Couldn't use `context.release_all` here due to borrow checker, but this does the same thing.